src/DataStructures/Tensor/EagerMath/OrthonormalOneform.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+// @{
+/*!
+ * \ingroup TensorGroup
+ *
+ * \brief Compute a spatial one-form orthonormal to the given unit form
+ *
+ * Given a unit spatial one-form \f$s_i\f$, compute a new form \f$t_i\f$
+ * which is orthonormal to \f$s_i\f$, in the sense that
+ * \f$\gamma^{ij}s_i t_j = 0\f$, for the given inverse spatial metric
+ * \f$\gamma^{ij}\f$.
```

Comment:
Can you document here how `t_i` is normalized? (even if just to say its norm is unspecified)